product_docs/docs/pem/9/registering_agent.mdx

```diff
@@ -247,4 +247,4 @@ To use a non-root user account to register a PEM agent, you must first install t
 
 !!! Note
     - Any probes and jobs that require root permission or access to a file owned by another user (for example,  enterprisedb) fail. 
-    - If you move the `agent.cfg` file from its default location to another, the PEM dashboard might display the agent status as “unknown”. See [Troubleshooting agent issues](troubleshooting_agent/#updating-configuration-file-path-agent-status-displaying-as-unknown), for more information. 
+    - If you move the `agent.cfg` file from its default location to another, the PEM dashboard might display the agent status as “unknown." See [Troubleshooting agent issues](troubleshooting_agent/#updating-configuration-file-path-agent-status-displaying-as-unknown), for more information. 
```

Comment:
```suggestion
    - If you move the `agent.cfg` file from its default location to another, the PEM dashboard might display the agent status as “unknown". See [Troubleshooting agent issues](troubleshooting_agent/#updating-configuration-file-path-agent-status-displaying-as-unknown), for more information. 
```
Typo?

Reply:
Are the quotes part of the string that displays? If not, in American English, the period always goes inside the quote. We generally do not use quote marks for this sort of case. I probably should have changed unknown to be tagged as code. Let me know if the quotes are part of the display, and I'll edit accordingly.

Reply:
@sonotley Does this work?

Reply:
Really? Weird. I'm not sure if it's an American/British thing but just from a basic logical point of view it doesn't seem to make sense to end the sentence inside the quotes. In any case, the current version looks fine and avoids the issue.

Reply:
@sonotley Oh, I hear ya, but it's true. If the quotes were actually part of the string, the period would indeed go outside. But in American English, comma and period otherwise *always* go inside the quotes--logical or not! Colon and semi-colon and likely other marks depend on context.